packages/mocaccino-dracut/build.yaml

```diff
@@ -0,0 +1,14 @@
+image: "quay.io/mocaccino/desktop:master"
+
+package_dir: /mocaccino-dracut
+
+prelude:
+- luet install sys-kernel/dracut
```

Comment:
You can safely drop this. In the build.yaml, the instructions are run to build the package. As you aren't actually running dracut in the steps, you don't need to install it